ocfweb/docs/docs/staff/procedures/deploying-services.md

```diff
@@ -0,0 +1,379 @@
+[[!meta title="Deploying Services"]]
+
+## Overview
+Are you looking to deploy a new service to the OCF Kubernetes cluster? This
+document will cover all of the steps required to do so. Do note that this is **not** a
+substitute for a Kubernetes tutorial or a Docker tutorial _(there are many
+resources online for that)_ but a guide for getting your service running on the
+OCF. There are a lot of pieces of OCF infrastructure at play here, but this
+should serve as a gentle introduction into how things are done.
+
+This `HOWTO` uses one of the OCF's simplest services as an example:
+[Templates][templates]. Templates is service used internally by OCF staff
+serving 'copy-pasteable' email templates. You can use [[git|doc
+staff/backend/git]] to `clone` this [repo](https://github.com/ocf/templates/blob/master/kubernetes/templates.yml.erb)
+, or just follow along looking at the files. If you want to blaze ahead,
+this repo should serve as a good example of the required components in the
+repo for deployment.
+
+## Prerequisites
+If you're looking to deploy your service, you should first have a service to
+deploy. This is normally done with a repository underneath the
+[OCF's Github organization](https://github.com/ocf). If you have the correct
+permissions, you should be able to transfer, fork, or just create a
+repository in the org.
+
+### Docker & Jenkins
```

Comment:
It would be better for people who want to deploy services to just clone or copy these files somehow. They're just going to copy paste from this doc anyway. I would wager that the vast majority of people who want to deploy services on kubernetes don't need to know how this works, it just has to "work." (of course, we should provide docs somewhere that explain how it works anyway)

You could have them run some script that takes in their proposed service name and it could even generate a basic template for those kubernetes yaml files too.

Probably out of scope for this PR, but something to consider for the future.

Reply:
Github has template repositories that can help solve the issue of copying files, so we can create one for Kubernetes deployments.